### DISCUSSION TITLE
Add a homebrew recipe to install ohmyzsh

### DISCUSSION THREAD

**Discussion by zabicki-stripe** [ohmyzsh/ohmyzsh#11941] [Ideas]
**Describe the solution you'd like**
It would be nice to be able to install ohmyzsh via homebrew. The alternative is to tell people to curl a shell script and execute it. (I know that installing via homebrew still poses risks, but at least homebrew provides an inventory of installed software)

**Comment by mcornella** [comment]
For that to happen OMZ would have to use some kind of version scheme. This was asked for some time in the past and rejected for this very same reason. Might happen in the near future though.

**Comment by Cyberbeni** [comment]
Can't you just `def install` and paste the proper usage of the install script there?

**Comment by robbyrussell** [comment]
Without versioning on our end, this is an awkward thing to handle.
